engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -493,6 +514,20 @@ public void setErrorMessage(String errorMessage)
       this.errorMessage = errorMessage;
     }
 
+    public String getLogFileName()
```

Comment:
Either introduce common parent that have logFileInfo or use an existing one. I don't see why this should be refactored later and we can't avoid code duplication now.